FastSimulation/Tracking/src/SeedFinderSelector.cc

```diff
@@ -27,6 +26,10 @@ SeedFinderSelector::SeedFinderSelector(const edm::ParameterSet & cfg,edm::Consum
     , eventSetup_(nullptr)
     , measurementTracker_(nullptr)
     , measurementTrackerLabel_(cfg.getParameter<std::string>("measurementTracker"))
+    , layerList(cfg.getParameter<std::vector<std::string>>("layerList"))
+    , layerPairs(cfg.getParameter<std::vector<unsigned>>("layerPairs"))
+    , BPix(cfg.getParameter<edm::ParameterSet>("BPix"))
```

Comment:
The layer configuration for BPix and FPix could not be set by this method. I'm investigating it but it would be great to have some suggestions.

Reply:
I didn't quite understand what you mean, could you elaborate? The `BPix` member is an `edm::ParameterSet`, so there should be no problems is copy-constructing that here.

But why do you need the `layerList`, `layerPairs`, `BPix`, and `FPix` member variables? At the moment they are not used (and the `layerPairs` parameter is actually read to `layerPairs_` member variable here
https://github.com/cms-sw/cmssw/blob/230f31097d63e124ba340ecfc1662e55a15a65f3/FastSimulation/Tracking/src/SeedFinderSelector.cc#L46-L52

Reply:
Yes I do not need the new parameters as member variables. I have removed them.